tests/common/test_cgroupapi.py

```diff
@@ -222,6 +223,42 @@ def test_cleanup_old_cgroups_should_move_daemon_pid_on_all_controllers(self):
         self.assertFalse(os.path.exists(old_cpu_cgroup))
         self.assertFalse(os.path.exists(old_memory_cgroup))
 
+        # Assert the event parameters that were sent out
+        _, kwargs = mock_add_event.call_args_list[0]
+        self.assertEquals(kwargs['op'], 'CGroupsCleanUp')
+        self.assertEquals(kwargs['is_success'], True)
+        self.assertEquals(kwargs['message'], 'Successfully cleaned up old cgroups in WALinuxAgent/WALinuxAgent.')
+
+    def test_cleanup_old_cgroups_should_fail_to_move_daemon_pid_on_all_controllers(self):
```

Comment:
In this naming convention the phrase after "should" should be the behavior that is being tested/asserted. In this case we are not testing that clean_old_cgroups fails to move the daemon pid on all controllers, but we are testing that it reports the errors from all controllers. A better name could be "test_cleanup_old_cgroups_should_report_errors_from_all_controllers_that_failed"

Reply:
Thanks, Norberto. This is what happens when you don't take time to think about naming :)